components/core/tools/scripts/lib_install/manylinux_2_28/install-prebuilt-packages.sh

```
@@ -13,33 +13,5 @@ dnf install -y \
-    --location \
-    --output "$task_pkg_path" \
-    --show-error \
-    "https://github.com/go-task/task/releases/download/v3.42.1/task_linux_${task_pkg_arch}.rpm"
-dnf install --assumeyes "$task_pkg_path"
-rm "$task_pkg_path"
-
-# Downgrade to CMake v3 to work around https://github.com/y-scope/clp/issues/795
-pipx uninstall cmake
-pipx install cmake~=3.31
+# Install `cmake`, `go-task` and `uv`
+"${script_dir}/../pipx_install/install-all.sh"
```

Shellcheck (0.10.0)

[warning] script_dir is referenced but not assigned.

(SC2154)